src/chip.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {createUseStyles} from './jss';
+import {useTheme} from './hooks';
+import Box from './box';
+import {Text2} from './text';
+import IconButton from './icon-button';
+import IconCloseRegular from './generated/mistica-icons/icon-close-regular';
+
+const useStyles = createUseStyles(({colors}) => ({
+    container: {
+        display: 'inline-flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: `1px solid ${colors.border}`,
+        borderRadius: 20,
+        backgroundColor: colors.backgroundContainer,
+    },
+}));
+
+type ChipProps = {
+    children: string;
+    icon?: React.ReactNode;
+    onClose?: () => void;
+};
+
+const Chip: React.FC<ChipProps> = ({children, icon, onClose}) => {
+    const classes = useStyles();
+    const {colors, texts} = useTheme();
+
+    return (
+        <Box
+            className={classes.container}
+            paddingY={4}
+            paddingLeft={icon ? 8 : 12}
+            paddingRight={onClose ? 8 : 12}
+        >
+            {icon && <Box paddingRight={4}>{icon}</Box>}
+            <Text2 medium>{children}</Text2>
+            {onClose ? (
+                <Box paddingLeft={8}>
```

Comment:
Change paddingLeft from 8 to 4, please, with the iconButton new size (24px) the icon is to far from label
![image](https://user-images.githubusercontent.com/6722153/149976759-aff79abd-2a8a-46ff-ba3e-e59866d63257.png)